Example/Harmony/Core/Features/Item/ItemProvider.swift

```diff
@@ -17,14 +16,14 @@ class ItemDefaultModule: ItemComponent {
     private let executor: Executor
     private let storage: AnyDataSource<Data>
     
-    init(executor: Executor, apiClient: Session, storage: AnyDataSource<Data>) {
+    init(executor: Executor, apiClient: URLSession, storage: AnyDataSource<Data>) {
         self.executor = executor
         self.storage = storage
     }
     
     private lazy var networkDataSource: AnyDataSource<ItemEntity> = {
-        let baseDataSource = GetNetworkDataSource<ItemEntity>(url: "https://demo3068405.mockable.io/", session: Session.default, decoder: JSONDecoder())
-        
+        let baseDataSource = GetNetworkDataSource<ItemEntity>(url: URL(string:"https://demo3068405.mockable.io/")!, session: URLSession.shared, decoder: JSONDecoder())
```

Comment:
You're doing a reference here to a `URLSession.shared` object, but you have in ApplicatoinDefaultModule, an `apiClient: URLSession` object that you instantiate there. Shouldn't it be that one instead?

Reply:
Yes, done.